docs/assistant/security-assistant.asciidoc

```diff
@@ -9,7 +9,7 @@
 
 The Elastic AI Assistant utilizes generative AI to bolster your cybersecurity operations team. It allows users to interact with {elastic-sec} for tasks such as alert investigation, incident response, and query generation or conversion using natural language and much more.
 
-A connector for OpenAI or Azure OpenAI Service powers AI Assistant.
+AI Assistant relies on Generative AI connectors to third-party AI providers.
```

Comment:
Should this be "to connect to"? For example:

```suggestion
AI Assistant relies on Generative AI connectors to connect to third-party AI providers.
```

Reply:
I was wondering the same exact thing, but I thought "connectors to connect" might be a confusing amount of "connect"ing. Anyone else want to weigh in?

Reply:
Hm, good poinT - "connect" doesn't add a whole lot of clarity. Wdyt about "interact" or "communicate"?